<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProject</class>
 <widget class="QDialog" name="NewProject">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_1">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_name">
           <property name="text">
            <string>Project Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="projectName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_base">
           <property name="text">
            <string>Project Base</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_base">
           <item>
            <widget class="QListWidget" name="pluginsList">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="showDropIndicator" stdset="0">
              <bool>false</bool>
             </property>
             <property name="currentRow">
              <number>-1</number>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>Choose one...</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QToolButton" name="openPluginsButton">
               <property name="minimumSize">
                <size>
                 <width>130</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>130</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Open Plugins Folder...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="aboutPluginButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>130</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>130</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>About Plugin</string>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
               <property name="autoRaise">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_location">
           <property name="text">
            <string>Project Location</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="projectDir">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
               <property name="clearButtonEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="locationButton">
               <property name="text">
                <string>Choose...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="projectDirFull">
             <property name="styleSheet">
              <string notr="true">color:rgb(128, 128, 128)</string>
             </property>
             <property name="text">
              <string>path/to/project</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="errorLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(255, 81, 69)</string>
         </property>
         <property name="text">
          <string>error_label</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="openExistingButton">
           <property name="text">
            <string>Open Existing Project...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="buttonBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="page_2_verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="setup_label">
         <property name="text">
          <string>Setting up project...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="progress_label">
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(127, 127, 127);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="finishButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>projectName</tabstop>
  <tabstop>pluginsList</tabstop>
  <tabstop>openPluginsButton</tabstop>
  <tabstop>projectDir</tabstop>
  <tabstop>locationButton</tabstop>
  <tabstop>openExistingButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewProject</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>574</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewProject</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>574</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>locationButton</sender>
   <signal>clicked()</signal>
   <receiver>NewProject</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>projectDir</sender>
   <signal>textChanged(QString)</signal>
   <receiver>NewProject</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>finishButton</sender>
   <signal>accepted()</signal>
   <receiver>NewProject</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openExistingButton</sender>
   <signal>clicked()</signal>
   <receiver>NewProject</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openPluginsButton</sender>
   <signal>clicked()</signal>
   <receiver>NewProject</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pluginsList</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>NewProject</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>projectName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>NewProject</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aboutPluginButton</sender>
   <signal>clicked()</signal>
   <receiver>NewProject</receiver>
   <slot>open_about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open_about()</slot>
 </slots>
</ui>
